.github/workflows/build-test-ci.yml

```diff
@@ -0,0 +1,46 @@
+name: log-exploration-oc-plugin build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+
+  build:
+    name: oc-plugin-build
+    runs-on: ubuntu-20.04
+    
+    steps:
+    - name: Git Checkout
+      uses: actions/checkout@v2
+
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16.3' # The Go version to download (if necessary) and use.
+    
+    - name: Install dependencies
+      run: |
+        go mod download
+        
+    - name: Run Unit tests
+      run: |
+        go test -race -covermode atomic -coverprofile=covprofile ./pkg/...
```

Comment:
@sangupta1 please use our Makefile target: `make test-unit`

Reply:
Sure @igor-karpukhin . I will make the change.